lib/dynflow/rails/configuration.rb

```diff
@@ -158,7 +158,7 @@ def queues
       protected
 
       def default_sequel_adapter_options(world)
-        db_config            = ::ActiveRecord::Base.configurations[::Rails.env].dup
+        db_config            = ::Rails.application.config.database_configuration[::Rails.env].dup
         db_config['adapter'] = db_config['adapter'].gsub(/_?makara_?/, '')
```

Comment:
In case someone is interested why something that is just deprecated does not work anymore:

Although `::ActiveRecord::Base.configurations[]` is deprecated in 6.1, it should still work. What silently changed however, is that the resulting hash has symbols as keys, whereas previously those were strings.

The new way with `::Rails.application.config.database_configuration[]` seems to work, the resulting hash uses strings as keys.